packages/opentelemetry/src/sampler.ts

```diff
@@ -50,11 +52,11 @@ export class SentrySampler implements Sampler {
       return wrapSamplingDecision({ decision: undefined, context, spanAttributes });
     }
 
-    // If we have a http.client span that has no local parent, we never want to sample it
+    // If we have a http.client or undici span that has no local parent, we never want to sample it
```

Comment:
IMHO we do not need to amend the comment here, because it is still a `http.client` span :D. it just comes from undici vs. http, but actually this here is generic and may also work for other instrumentation!